src/screens/MemberDetail/MemberDetail.tsx

```diff
@@ -74,252 +226,363 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
     navigate(`/orgpeople/${currentUrl}`);
   }
 
-  const addAdmin = async (): Promise<void> => {
-    try {
-      const { data } = await adda({
-        variables: {
-          userid: location.state?.id,
-          orgid: orgId,
-        },
-      });
-
-      /* istanbul ignore next */
-      if (data) {
-        try {
-          const { data } = await updateUserType({
-            variables: {
-              id: location.state?.id,
-              userType: 'ADMIN',
-            },
-          });
-          if (data) {
-            toast.success(t('addedAsAdmin'));
-            setTimeout(() => {
-              window.location.reload();
-            }, 2000);
-          }
-        } catch (error: any) {
-          errorHandler(t, error);
-        }
-      }
-    } catch (error: any) {
-      /* istanbul ignore next */
-      if (
-        userData.user.userType === 'ADMIN' ||
-        userData.user.userType === 'SUPERADMIN'
-      ) {
-        if (isMounted.current) setIsAdmin(true);
-        toast.error(t('alreadyIsAdmin'));
-      } else {
-        errorHandler(t, error);
-      }
-    }
-  };
+  const sanitizedSrc = sanitizeHtml(formState.file, {
```

Comment:
Are we previewing the uploaded image ?, is that why you have added this package here ?, Because sanitization of uploaded files is generally done on the server.

Reply:
> Are we previewing the uploaded image ?, is that why you have added this package here ?, Because sanitization of uploaded files is generally done on the server.

Yes, I've made the formState in such a way to preview things like `firstName` and `avatar` on the right box
<img width="429" alt="image" src="https://github.com/PalisadoesFoundation/talawa-admin/assets/76090263/27d651a1-1760-4241-bfd6-9adc1d1f06dc">

Reply:
@pranshugupta54 so you are saying that on clicking on the Choose File button, the image that you have selected will be visible at Profile Details card ?

Reply:
@rishav-jha-mech, yes. Also if you change `firstName` in the input box, it will show in the Profile Details card. 
Now if you click "Save", it saves otherwise it's lost

Reply:
Example: https://www.loom.com/share/959a80f3602b4894bf8acc7bf5e59161?sid=025def3a-4b56-426d-a6d3-2de1c5d67f0e